src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -303,9 +303,9 @@ bool fx_muxer_t::resolve_hostpolicy_dir(host_mode_t mode,
         return true;
     }
 
-    // If it still couldn't be found, flag an error for the "expected" location.
-    trace::error(_X("Expect required library %s to be present in [%s]"), LIBHOSTPOLICY_NAME, expected.c_str());
-    trace::error(_X("  - This may be because of an invalid .NET Core FX configuration in the directory."));
+    // If it still couldn't be found, somebody upstack messed up. Flag an error for the "expected" location.
+    trace::error(_X("The library '%s' required to execute the application was not found in '%s'"), LIBHOSTPOLICY_NAME, expected.c_str());
+    trace::error(_X("  - This is a fatal error, and an invalid application configuration in the above directory."));
```

Comment:
This is just a fatal error with some files missing, so somebody messed up the configuration. Do you have any suggestions for rewording it? I'm inclined to making it "Fatal error: The library %s is required..."